providers/implementations/rands/drbg_ctr.c

```diff
@@ -85,6 +85,8 @@ static void ctr_XOR(PROV_DRBG_CTR *ctr, const unsigned char *in, size_t inlen)
      * are XORing. So just process however much input we have.
      */
     n = inlen < ctr->keylen ? inlen : ctr->keylen;
+    if (n > sizeof(ctr->K))
```

Comment:
Given these are returns from void functions and they are basically _impossible_ conditions, I'd suggest using `!ossl_assert(n <= sizeof(ctr->K))` instead.

Reply:
updated and force pushed.